src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -311,6 +311,22 @@ def _token_to_char(token):
                 answer=example.context_text[char_start[0] : char_end[1]],
             )
 
+    @staticmethod
+    def route_input_to_bucket(
+        *args, input_schema: BaseModel, pipelines: List[TransformersPipeline], **kwargs
+    ) -> Pipeline:
+        """
+        :param input_schema: The schema representing an input to the pipeline
+        :param pipelines: Different buckets to be used
+        :return: The correct Pipeline object (or Bucket) to route input to
+        """
+        current_seq_len = len(input_schema.question.split())
```

Comment:
is all of the "splitting" a concern since tokenization might produce more tokens than just words?

Reply:
good call out, splitting was originally put in to get MVP of this functionality quickly working. definitely need to discuss a better strategy

Reply:
As synced offline, relying on `split` right now, tokenization for `should_bucket` will be made better in the next sprint